scripts/fetch-asyncapi-example.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const request = require('request');
+const fs = require('fs');
+const unzipper = require('unzipper');
+const path = require('path');
+const parser = require('@asyncapi/parser');
+
+const SPEC_EXAMPLES_ZIP_URL = 'https://github.com/asyncapi/spec/archive/refs/heads/master.zip';
+const EXAMPLE_DIRECTORY = path.join(__dirname, '../assets/examples');
+const TEMP_ZIP_NAME = 'spec-examples.zip';
+
+const fetchAsyncAPIExamplesFromExternalURL = () => {
+  return new Promise((resolve) => {
+    request(SPEC_EXAMPLES_ZIP_URL)
+      .pipe(fs.createWriteStream(TEMP_ZIP_NAME))
+      .on('close', () => {
+        console.log('Fetched ZIP file');
+        resolve();
+      });
```

Comment:
Should we add error handling here?

```suggestion
      })
      .on('error', () => {
        reject();
      });
```

*Note: not sure if this code is correct.*

Reply:
Thanks, will add it, yeah I believe it is from what I read about the `createReadStream` api, thanks!

Reply:
This is still not fixed (I'm saying because you asked me for review again 😄)

Reply:
Ah, thanks my bad 😅

Reply:
Ok, think it's done now, sorry lol, thought I made the change but looks like I did'nt